pipeline/pipeline_tasks/api_calls/00_01_yelp.py

```diff
@@ -5,34 +5,35 @@
 @author: User
 """
 
+import pprint
 import requests
 from bs4 import BeautifulSoup
 
-page = requests.get('https://maps.googleapis.com/maps/api/place/textsearch/json?query=El+Farolito+2779+Mission+Street+San+Francisco&key=AIzaSyD_xdgTbmgL-pVFykN-rMZIQF3DdeMszA0')
-soup = BeautifulSoup(page.content, 'html.parser')
-
-data = {'grant_type': 'client_credentials',
-        'client_id': 'kiD4d8ra_jebdWlH9XZyLw',
-        'client_secret': 'yAm96WC7WVyKhCUobTu0YX3nl5EqmlnVQnQrdRMomrMCDg4auhg7aK9QJyEIyUIm'}
-token = requests.post('https://api.yelp.com/oauth2/token', data=data)
-access_token = token.json()['access_token']
-url = 'https://api.yelp.com/v3/businesses/search'
-headers = {'Authorization': 'bearer %s' % access_token}
-params = {'location': 'San Bruno',
-          'term': 'Japanese Restaurant',
-          'pricing_filter': '1, 2',
-          'sort_by': 'rating'
-         }
-
-resp = requests.get(url=url, params=params, headers=headers)
-
-import pprint
-pprint.pprint(resp.json()['businesses'])
-
-
-
-
 
-page = requests.get('https://api.yelp.com/v3/businesses/search?term=delis&latitude=37.786882&longitude=-122.399972')
-soup = BeautifulSoup(page.content, 'html.parser')
+def main():
```

Comment:
hmm...i see what you mean...i think i threw this in there just to show the general code i used...i can write the clean script tomorrow - i found it in my files...